src/README.md

```diff
@@ -589,6 +589,70 @@ void InitializeHttpParser(Local<Object> target,
 }
 ```
 
+<a id="Argument validation in public APIs vs. internal code"></a>
+
+### Argument validation in public APIs vs. internal code
+
+#### Public API argument sanitization
+
+When arguments come directly from user code, Node.js validates them at the
```

Comment:
```suggestion
When arguments come directly from user code, Node.js will typically validate them at the
```

Softening just a bit given the fact (as called out further below) that there are places where we perform the type validation in c++